content/_index.md

```diff
@@ -7,10 +7,26 @@ BookToC: false
 
 # The Linux Userspace API (UAPI) Group
 
-The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.
+The userspace API ("uapi") group is a cross-distro community for people with an interest in innovating how we build, deploy, run, and safely update modern Linux operating systems.
 It serves as a central gathering place for specs, documentation, and ideas.
 
-Follow [@uapi-group](https://twitter.com/uapi_group) on twitter to catch the latest news!
+In scope of the group are secure boot mechanisms, kernel / disk image formats, TPM usage, safe and secure updates, and related topics.
+The group also cares about interop of image-based and/or immutable Linux distributions.
+
+The group was founded to discuss and align on common mechanisms, to prevent creating quasi standards by going with implementations first.
```

Comment:
An implementation might become a spec and the other way around. So I would rather say:

```
The uapi group aims to establish specifications for common concepts shared among relevant parties. We are open to anyone who has an interest in constructive collaboration!
```

or sm like this